proteus/mprans/RANS3PSed2D.h

```diff
@@ -1951,22 +1965,23 @@ namespace proteus
                                                   q_grad_vos[eN_k_nSpace+0],
                                                   q_grad_vos[eN_k_nSpace+1],
                                                   q_grad_vos[eN_k_nSpace+1]);
-		updatePenaltyForPacking(vos,
-					u,
-					v,
-					w,
-					mom_u_source,
-					mom_v_source,
-					mom_w_source,
-					dmom_u_source,
-					dmom_v_source,
-					dmom_w_source);
 
-				               updateFrictionalPressure(vos,
-                        grad_vos,
-						mom_u_source,
-						mom_v_source,
-						mom_w_source);      
+		/* /\* updatePenaltyForPacking(vos, *\/ */
```

Comment:
Agreed, remove this, not necessary if the limiter is around

Reply:
OK, will do.